release/unstable/clearlinux/docker/Dockerfile

```diff
@@ -13,7 +13,7 @@ ARG PS_PACKAGE_URL=https://github.com/PowerShell/PowerShell/releases/download/v$
 ARG PS_INSTALL_VERSION=7
 
 # Download the PowerShell Core Linux tar.gz and save it
-ADD ${PS_PACKAGE_URL} /tmp/powershell-linux.tar.gz
+RUN curl -sSL ${PS_PACKAGE_URL} -o /tmp/powershell-linux.tar.gz
```

Comment:
We can' use curl either :(

Reply:
need to remove ADD and curl from unstable channel images.

Reply:
We can ignore unstable.